l1nk4i/docs/apiDoc.md

```
@@ -0,0 +1,890 @@
+DELETE /api/answers/{answer-id}
+
+删除回答
+
+### 请求参数
+
+|名称|位置|类型|必选|说明|
+|---|---|---|---|---|
+|answer-id|path|string| 是 |none|
+
+> 返回示例
+
```

Markdownlint

Blank line inside blockquote

(MD028, no-blanks-blockquote)